core/src/main/scala/com/rallyhealth/vapors/core/math/MathSyntax.scala

```diff
@@ -0,0 +1,15 @@
+package com.rallyhealth.vapors.core.math
```

Comment:
BLKR: This file is duplicated in the syntax package. We should remove this file.